pkg/network/cache/client.go

```diff
@@ -30,7 +30,7 @@ func NewSDKClientCache() *ClientCache {
 // Get function returns existing client or creates a new one. Consider passing
 // connection options to specify details for client, but don't forget that two
 // different set of options should provide two different clients.
```

Comment:
Lets remove `Consider passing connection options...` comment now, since `Get` function now supports options.

Reply:
Fixed, thx. For different set of options we will use different `ClientCache` instances.